NickvisionMoney.Shared/Models/Account.cs

```diff
@@ -1322,7 +1323,10 @@ private async Task<List<uint>> ImportFromQIFAsync(string path, string rgba)
         QifDocument? qif = null;
         try
         {
-            qif = QifDocument.Load(File.OpenRead(path));
+            qif = Task.Run(() => {
```

Comment:
Task.Run is awaitable so you should be able to make it 
```csharp
qif = await Task.Run(() =>
{
    Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
    return QifDocument.Load(File.OpenRead(path));
});
```
and remove the need for `.Result`

Reply:
And actually, `Task.Run` and any `Task` for that matter isn't truly guaranteed to run in a separate thread. 
So it's probably best to just make it
```csharp
var oldCulture = Thread.CurrentThread.CurrentCulture;
Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
qif = QifDocument.Load(File.OpenRead(path));
Thread.CurrentThread.CurrentCulture = oldCulture;
```
within the try block itself.

Reply:
> And actually, `Task.Run` and any `Task` for that matter isn't truly guaranteed to run in a separate thread.

Ouch, I though it is, that's why I preferred to use it instead of changing the culture in current thread like you said. But I guess there's no better option

Reply:
> But I guess there's no better option

Yeah...i mean even if I were to send a patch the the library itself, i'd pretty much be the same code. Change the culture to english before, load the qif, change it back to previous. So...